Tasks/PyPIPublisher/publisher.ts

```
@@ -19,27 +17,27 @@ var password = tl.getEndpointAuthorizationParameter(serviceEndpointId, 'password
 //Create .pypirc file
 var text = util.format("[distutils] \nindex-servers =\n    pypi \n[pypi] \nrepository=%s \nusername=%s \npassword=%s", pythonServer, username, password);
 tl.writeFile(pypircFilePath, text, 'utf8');
-publishPythonPackage();
 
-async function publishPythonPackage(){
+async function run(){
     //PyPI upload
     try{
-        await executePythonTool("-m pip install wheel twine --user");
-        await executePythonTool(util.format("%s sdist --dist-dir %s bdist_wheel", setupFilePath, distDirectoryPath));    
-        await executePythonTool(util.format("-m twine upload %s", uploadArtifactsPath));    
+        tl.cd(workingDirectory);
+        await executePythonTool("-m pip install twine --user");
+        await executePythonTool("setup.py sdist");
+        await executePythonTool("-m twine upload dist/*");
     }
     catch(err){
-        tl.setResult(tl.TaskResult.Failed, err);
+        tl.setResult(tl.TaskResult.Failed, "Upload Failed");
```

This should be localized

---

Are you hiding the original error